tool/tctl/common/access_request_command.go

```diff
@@ -228,6 +234,18 @@ func (c *AccessRequestCommand) Delete(client auth.ClientI) error {
 	return nil
 }
 
+func (c *AccessRequestCommand) Caps(client auth.ClientI) error {
+	caps, err := client.GetAccessCapabilities(context.TODO(), services.AccessCapabilitiesRequest{
+		User:             c.user,
+		RequestableRoles: true,
+	})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	fmt.Printf("Requestable Roles: %+v\n", caps.RequestableRoles)
```

Comment:
This is inconsistent with how we typically return results, we usually return a table and support format flags like `--json`.